### PR TITLE
Add config files for mellanox-mlnxen and intel-e1000e

### PR DESCRIPTION
**Pull request by gduperrey**
Modify the cfg fil for intel-e1000e
Add the cfg file for mellanox-mlnxen

### CODE REVIEW

File: intel-e1000e-alt-8.2.cfg [hunk 1]
```diff
@@ -3,4 +3,4 @@ PACK_UUID="383ac20f-9c83-41be-85b0-9acf2aa3ab53"
 PACK_BUILD="pack.0.1"
 RPM_FILE="intel-e1000e-alt-3.8.7-1.xcpng8.2.x86_64.rpm"
 RPM_SHA256="4f3716c4bd5765b97f538b4f021d51853939df6505fc0a62aa718d532c64c5da"
-GPG_PUBKEY_FILE="RPM-GPG-KEY-XS-DDK-Test"
+GPG_PUBKEY_FILE="RPM-GPG-KEY-XS-DDK-SingleUse"
```
Comment (stormi) [inline intel-e1000e-alt-8.2.cfg]: We probably don't need XS DDK in the key name.

```suggestion
GPG_PUBKEY_FILE="RPM-GPG-KEY-XCP-SingleUse"
```

File: mellanox-mlnxen-alt-8.2.cfg [hunk 2]
```diff
@@ -0,0 +1,6 @@
+PACK_DESC="Driver Pack for Intel e1000e device driver"
```
Comment (stormi) [inline mellanox-mlnxen-alt-8.2.cfg]: Wrong description, this is mellanox mlnxen.

**Review by stormi — CHANGES_REQUESTED** [review]
The commit title says "add" but only one is added, the other one is modified. 

Ideally, there should be one commit modifying the key name in existing files, and another commit which adds the new one.

**Comment by gduperrey** [comment]
And here is it with two separate commits.

**Review by stormi — APPROVED** [review]

**Pull request merged**